x-pack/plugins/rule_registry/server/rule_registry/rule_type_helpers/create_lifecycle_rule_type_factory.ts

```diff
@@ -68,6 +68,13 @@ export function createLifecycleRuleTypeFactory(): CreateLifecycleRuleType<BaseRu
           rule,
         } = options;
 
+        const so = await options.services.savedObjectsClient.get<AlertAttributes>(
+          'alert',
+          rule.uuid
+        );
+
+        console.error('RULE REGISTRY CONSUMER', so.attributes.consumer);
```

Comment:
Currently the `consumer` field is not available via params in the executor so we have to query for the rule and then acquire the consumer off of the rule's saved object attributes.

ref: https://github.com/elastic/kibana/pull/98935#issuecomment-832191175